library_website/templates/404.html

```diff
@@ -1,21 +1,144 @@
-{% extends "base/public_base.html" %}
+{% extends extention %}
+{% load wagtailimages_tags %}
 
 {% block body_class %}template-404{% endblock %}
 
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+</head>
```

Comment:
@bbusenius isn't there another place in the Wagtail site where we usually stick jQuery, Bootstrap, etc. or am I misremembering?  One of the includes in the `base` app, maybe?

Reply:
Yes, normally this is inherited from the respective base templates. This was something that struck me too. I'm guessing the reason for it is that the 404 templates sit outside of the infrastructure (they're not Wagtail page objects)? I was going to look at this closer tomorrow.

Reply:
Actually, I think all of the wrapper html is extraneous and unnecessary. That includes the `html`, `head`, and `body` tags. From what I can tell, most of this isn't being included in the rendered html, except for one of the `body` close tags (which means there are two and it's producing invalid html). That's because the template won't use anything that isn't inside one of the blocks: `{% block content %}` and `{% block body_class %}` in this case. So, everything that's outside of `{% block content %}` and `{% block body_class %}`  should be eliminated. More on template inheritance here: https://docs.djangoproject.com/en/4.1/ref/templates/language/#template-inheritance